pipeline/common_stages.groovy

```diff
@@ -390,7 +390,9 @@ def deploy_mig_controller_on_both(
     stage('Build mig-controller image and deploy on both clusters') {
       steps_finished << 'Build mig-controller image and deploy on both clusters'
       // Create custom mig-controller docker image if building a different mig-controller repo/branch
-      if ("${MIG_CONTROLLER_REPO}" != "https://github.com/konveyor/mig-controller.git") {
+      if ("${MIG_CONTROLLER_REPO}" != "https://github.com/konveyor/mig-controller.git" || 
+          "${MIG_CONTROLLER_BRANCH}" != "master"
+        ) {
```

Comment:
If we want to build a controller image with konveyor/mig-controller repo but with different branch than master  `konveyor/mig-controller:release-1.1`